web-ui/src/main/resources/catalog/js/edit/EditorController.js

```diff
@@ -570,8 +570,8 @@
                 closeEditor();
               }, function(reason) {
                 $rootScope.$broadcast('StatusUpdated', {
-                  title: $translate.instant(reason.data.error.message),
-                  error: reason.data.error.description,
+                  title: reason.data.message, //returned error JSON obj
+                  error: reason.data.description,
```

Comment:
To confirm: has the code been reviewed to verify `reason.data.error.message` and `reason.data.error.description` are not used?

Otherwise, I would used the new properties as a fallback if the original ones are not returned.

Reply:
@josegar74 

The "reason.data.error.message" and "reason.data.error.description" are not used for this Angular module. I don't think it's a property but it's a JSON returned from the API side.